litmus-portal/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -228,6 +228,31 @@ func CreateChaosWorkflow(input *model.ChaosWorkFlowInput, r store.StateData) (*m
 	}, nil
 }
 
+func UpdateWorkflow(workflow *model.ChaosWorkflowUpdateInput, r store.StateData) (*model.ChaosWorkFlowResponse, error) {
+	query := bson.D{{"cluster_id", workflow.WorkflowID}}
+	update := bson.D{{"$set", bson.D{{"project_id", workflow.ProjectID}, {"cluster_id", workflow.ClusterID}, {"workflow_manifest", workflow.WorkflowManifest}, {"cronSyntax", workflow.CronSyntax}, {"workflow_name", workflow.WorkflowName}, {"workflow_description", workflow.WorkflowDescription}, {"isCustomWorkflow", workflow.IsCustomWorkflow}, {"weightages", workflow.Weightages}, {"updated_at", strconv.FormatInt(time.Now().Unix(), 10)}}}}
```

Comment:
Project_id cannot and  cluster_id cannot and should not change

Reply:
Only workflow content should be updated

Reply:
Done